### PR TITLE
Fixes selections

### LINKED ISSUE #95: Selection from the tree view seems broken now?
I am not able to select from the tree view anymore:

- it does not update the selection in the 3D view
- it does not show the current selection in the tree view

### PR DESCRIPTION
**Pull request by hbcarlos**
Fixes #95 

https://user-images.githubusercontent.com/26092748/213879794-1a0287e9-315d-48d6-a350-79c77e4c6233.mov

### CODE REVIEW

**Review by martinRenou — APPROVED** [review]
Nice refactoring 👍🏽

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,7 @@ jupytercad/labextension
 
 # Created by https://www.gitignore.io/api/python
 # Edit at https://www.gitignore.io/?templates=python
+jupytercad/_version.py
```
Comment (martinRenou) [inline .gitignore:11]: What is the reason for adding this to gitignore?

Comment (martinRenou) [reply .gitignore:11]: I suppose it's related to the hatch migration?

**Review by trungleduc — APPROVED** [review]

**Comment by trungleduc** [comment]
Thanks @hbcarlos

**Comment by martinRenou** [comment]
I see a bit of latency in zooming/panning introduced by this PR. I'm having a look.

**Pull request merged**